src/services/export.service.js

```diff
@@ -208,17 +254,45 @@ function validateExportParams(request, reply) {
   return true;
 }
 
+/**
+ * Pulls query parameters from both the url query and request body and creates a new parameters map
```

Comment:
I have a question about the POST requests for group-level and patient-level exports. Correct me if I am wrong, but I believe the current functionality allows POST requests for group-level and patient-level exports with query params from either the request body or the url query. In the spec, it says "A server SHALL support GET requests and MAY support POST requests that supply parameters using the FHIR Parameters Resource" (https://www.hl7.org/fhir/parameters.html)." (https://build.fhir.org/ig/HL7/bulk-data/export.html#bulk-data-kick-off-request) Does this mean that POST requests should only be supported with query params in the request body as a FHIR Parameters resource? If so, we may want to add that check and throw an error like we do for the patient param in GET requests.

Reply:
Good catch - yes, I think we should assume that POST requests required a FHIR Parameters Resource as the request body for parameters if any params are specified. I will look into adding checks for the parameters being supplied in the body rather than in the url

Reply:
I added a check for parameters being supplied in the url, and a check for the request body to be of type "Parameters."  Open to adding additional handling here as well if any other edge cases come up while testing!